osu.Game/Overlays/BeatmapSet/Scores/ScoreTable.cs

```diff
@@ -65,7 +67,9 @@ public IReadOnlyList<ScoreInfo> Scores
                 for (int i = 0; i < value.Count; i++)
                     backgroundFlow.Add(new ScoreTableRowBackground(i, value[i], row_height));
 
-                Columns = createHeaders(value[0]);
+                isBeatmapRanked = value.First().Beatmap.Status == BeatmapSetOnlineStatus.Ranked;
```

Comment:
This is not entirely what I was looking for. There is nothing stopping users from providing a list of scores from different beatmaps (while there's no situation I can think of right now that would necessitate that, it isn't out of the question), at which point the `First()` would be wrong. I'd honestly prefer a plain, explicit ctor parameter or property.

Reply:
Fair point, I haven't thought about that scenario at all. Will change.